release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -545,6 +553,9 @@ For more information, see xref:../operators/understanding/olm-packaging-format.h
 ==== Java-based Operators (Technology Preview)
 
 Starting in {product-title} {product-version} as a Technology Preview feature, the Operator SDK includes the tools and libraries to develop a Java-based Operator. Operator developers can take advantage of Java programming language support in the Operator SDK to build a Java-based Operator and manage its lifecycle.
+In earlier versions of {product-title}, the Performance Addon Operator provided automatic, low latency performance tuning for applications. In {product-title} 4.11, these functions are part of the Node Tuning Operator. The Node Tuning Operator is part of the standard installation for {product-title} 4.11. If you upgrade to {product-title} 4.11, the Node Tuning Operator removes the Performance Addon Operator and all related artifacts on startup.
```

Comment:
Is this in the right place? Seems to be in between Java based operators and the link to those.

Reply:
I will investigate. It's not my content - at first glance looks like something that happened during merge conflict resolution.

Reply:
Resolved the restructuring issue: merge conflict.